From 194af16100dee886ceef175fab896dc91127ffab Mon Sep 17 00:00:00 2001
From: Grzegorz Jaszczyk <jaz@semihalf.com>
Date: Wed, 25 Apr 2018 16:36:05 +0200
Subject: [PATCH 0618/1051] phy: cp110-comphy: power off all comphy during
 probe

To avoid relying on the bootloader/firmware configuration, power off all
comphys during comphy driver probe.

Change-Id: I18885923aac42a1f1da811d2f5e1f133568ad864
Signed-off-by: Grzegorz Jaszczyk <jaz@semihalf.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/phy/marvell/phy-mvebu-cp110-comphy.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/phy/marvell/phy-mvebu-cp110-comphy.c b/drivers/phy/marvell/phy-mvebu-cp110-comphy.c
index ed3ee78f7d09..f4d27b8e2ac5 100644
--- a/drivers/phy/marvell/phy-mvebu-cp110-comphy.c
+++ b/drivers/phy/marvell/phy-mvebu-cp110-comphy.c
@@ -295,10 +295,10 @@ static int mvebu_comphy_probe(struct platform_device *pdev)
 		phy_set_drvdata(phy, lane);
 
 		/*
-		 * Once all modes are supported in this driver we should call
-		 * mvebu_comphy_power_off(phy) here to avoid relying on the
-		 * bootloader/firmware configuration.
+		 * To avoid relying on the bootloader/firmware configuration,
+		 * power off all comphys.
 		 */
+		mvebu_comphy_power_off(phy);
 	}
 
 	dev_set_drvdata(&pdev->dev, priv);
-- 
2.17.1

